docs/scenarios_authoring/DataSourcesAndSinks.md

```diff
@@ -34,8 +34,23 @@ The `kafka` sink configuration form will show a list of fields defined in Schema
 
 ![Kafka sink](img/kafkaSink.png "Kafka sink")
 
-If you prefer to define the whole output event as one expression containing value to be written to the Kafka topic, instead of filling separete autogenerated fields, you can do that by switching `Raw editor` to `true`.
+If you prefer to define the whole output event as one expression containing value to be written to the Kafka topic, instead of filling separate autogenerated fields, you can do that by switching `Raw editor` to `true`.
 
 When using the Raw editor, you can choose the validation mode:
-* strict - requires providing all fields, including optional fields, without redundant fields;
-* lax - requires providing only required fields, you can skip optional fields and add redundant fields.
\ No newline at end of file
+* strict - requires providing all fields, including optional fields, without redundant fields, requires exact matching;
```

Comment:
add something like (*) to this _strict_  term to indicate that it is explained later. Of course I immediately started to wonder what it means and why it is not explained here.